CDP4ShellDialogs/Views/ModelOpeningDialog.xaml

```diff
@@ -34,123 +32,149 @@
+                              DisplayMember="Name"></dxe:ComboBoxEdit>
+        </dxlc:LayoutItem>
         <dxlc:LayoutGroup HorizontalAlignment="Stretch"
                           VerticalAlignment="Stretch"
-                          Header="Engineering Models"
+                          Header="{Binding ScreenTitle}"
                           View="GroupBox"
-                          Grid.Row="0"
+                          Grid.Row="1"
                           Margin="5"
                           Orientation="Vertical">
-            <dxg:TreeListControl x:Name="iterationSelectionTreeListControl"
-                                 HorizontalAlignment="Stretch"
-                                 VerticalAlignment="Stretch"
-                                 ItemsSource="{Binding SessionsAvailable}"
-                                 SelectedItems="{Binding SelectedIterations}"
-                                 SelectionMode="Row" >
-                <dxg:TreeListControl.Columns>
-                    <dxg:TreeListColumn FieldName="Name" HorizontalHeaderContentAlignment="Left"/>
-                    <dxg:TreeListColumn CellTemplateSelector="{StaticResource DoETemplateSelector}"
-                                        FieldName="SelectedDomain"
-                                        Header="Domain Of Expertise"
-                                        HorizontalHeaderContentAlignment="Left"
-                                        AllowEditing="True" />
-                    <dxg:TreeListColumn FieldName="FrozenOnDate" Header="Frozen On"  HorizontalHeaderContentAlignment="Left"/>
-                </dxg:TreeListControl.Columns>
-                <dxg:TreeListControl.View>
-                    <dxg:TreeListView x:Name="View"
-                                      AllowEditing="False"
-                                      AutoExpandAllNodes="True"
-                                      AutoWidth="True"
-                                      HorizontalScrollbarVisibility="Auto"
-                                      NavigationStyle="Cell"
-                                      ShowHorizontalLines="False"
-                                      ShowIndicator="False"
-                                      ShowNodeImages="False"
-                                      ShowVerticalLines="False"
-                                      TreeDerivationMode="HierarchicalDataTemplate"
-                                      TreeLineStyle="Solid"
-                                      VerticalScrollbarVisibility="Auto"/>
-                </dxg:TreeListControl.View>
+            <dxg:GridControl Visibility="{Binding IsModelScreen, Converter={dx:BooleanToVisibilityConverter}}"
+                         Name="ModelSetupGridControl"
+                         AllowLiveDataShaping="False"
+                         ItemsSource="{Binding SelectedRowSession.EngineeringModelSetupRowViewModels}"
+                         SelectedItem="{Binding SelectedRowModel,
```

Comment:
`SelectedRowModel` should be renamed to reflect that its talking about `EngineeringModel` rows.